app/Resources/views/default/index.html.twig

```diff
@@ -19,7 +19,7 @@
             <div id="next">
                 <h2>What's next?</h2>
                 <p>
-                    <svg id="icon-book" version="1.1" xmlns="http://www.w3.org/2000/svg" x="0px" y="0px" viewBox="-12.5 9 64 64" enable-background="new -12.5 9 64 64" xml:space="preserve">
+                    <svg id="icon-book" version="1.1" xmlns="https://www.w3.org/2000/svg" x="0px" y="0px" viewBox="-12.5 9 64 64" enable-background="new -12.5 9 64 64" xml:space="preserve">
```

Comment:
Ah, I followed the link and was redirected to the `https`, that's why I decided to change it.

Thanks, I'll revert it!